### PR TITLE
new variant: PRNTRv2

### PR DESCRIPTION
**Pull request by ghent360**
**Summary**

New 3D printer board variant PRNTR board V2. Design files at
https://github.com/ghent360/PrntrBoardV2

This PR fixes/implements the following **features**
* Add support for my custom 3D printer controller board.

**Validation**
Tested Marlin bugfix-2.0.x branch. Tested HardwareTimers, HardwareSerial, USB CDC, SD-card SDIO, I2C (TwoWire), SPI, GPIO, ADC, Servo

### CODE REVIEW

File: variants/PRNTR_V2/variant.h [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+#define PA15   79
+
+// This must be a literal
+#define NUM_DIGITAL_PINS        80
+// This must be a literal with a value less than or equal to to MAX_ANALOG_INPUTS
+#define NUM_ANALOG_INPUTS       8
+#define NUM_ANALOG_FIRST        64
+
+// On-board LED pin number
+#define LED_BUILTIN             -1
```
Comment (fpistm) [inline variants/PRNTR_V2/variant.h]: If no LED BUILTIN I guess you could simply comment or remove it.

Comment (ghent360) [reply variants/PRNTR_V2/variant.h]: removed

**Comment by fpistm** [comment]
@ghent360 
One question:
maybe it should be fine to merge the two PRNTR and use switch for the difference?

**Comment by ghent360** [comment]
Sure how do I do that?

**Comment by fpistm** [comment]
> 
> Sure how do I do that?

I do the same for https://github.com/stm32duino/Arduino_Core_STM32/pull/828

In fact you can use the switch : `ARDUINO_<board>`
In your case:
`ARDUINO_PRNTR_F407_V1`
or
`ARDUINO_PRNTR_V2`

and in the boards.txt point to the same variant folder.

**Comment by ghent360** [comment]
ok. I'll update it. Give me a few days.

**Comment by ghent360** [comment]
merged into a single variant.

**Pull request merged**